test/web-platform-tests/to-upstream/dom/nodes/move-me-plz.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
I need a bit of guidance on:

1. where to move the test files
2. how to split them up

https://github.com/w3c/web-platform-tests has a ton of tests :joy: 

All the new tests are focused on click events for either HTMLElement or HTMLInputElement.

This actually looks pretty close to the tests I have now https://github.com/w3c/web-platform-tests/blob/master/DOMEvents/tests/approved/dispatchEvent.click.checkbox.html